Python/Quantum_random/README.md

```diff
@@ -0,0 +1,23 @@
+# Quantum random number generation
+This python script explains how classical computations can be done using Quantum gates. It generates a random number using Quantum computation.
+
+### How to use?
+
+- Intially register at [IBM quantum computing](https://quantum-computing.ibm.com/) go to my account section located in top-right where you will find your credentials and information on how to run Qiskit package locally.
+
+- Additionally, I have included a section in comments for you to save your account. I recommend to setup a virtual environment for it, so you can easily uninstall packages if you do not want to use them further.
+
+![Qiskit API](https://i.imgur.com/xJ3RVX5.png)
+
+### Just type in:
+
+`python Quant_random.py`
+
+### Requirements
+In case you want to setup a virtual environment. Read [this](https://realpython.com/python-virtual-environments-a-primer/).
+
+Before running the script, just install qiskit for python using this command:
+
+`pip install qiskit`
+
```

Comment:
Can a snap be added on what does the script yields?

Reply:
Sure!